src/pages/settings/Security/TwoFactorAuth/VerifyPage.js

```diff
@@ -0,0 +1,151 @@
+     *
+     * @returns {string}
+     */
+    function buildAuthenticatorUrl() {
+        return `otpauth://totp/Expensify:${props.account.primaryLogin}?secret=${props.account.twoFactorAuthSecretKey}&issuer=Expensify`;
+    }
+
+    return (
+        <ScreenWrapper>
+            <HeaderWithCloseButton
+                title={props.translate('twoFactorAuth.headerTitle')}
+                shouldShowStepCounter
+                stepCounter={{
+                    step: 2,
+                    text: props.translate('twoFactorAuth.stepVerify'),
+                }}
+                shouldShowBackButton
+                onBackButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_2FA_CODES)}
+                onCloseButtonPress={() => Navigation.dismissModal(true)}
+            />
+            <FullPageOfflineBlockingView>
+                <ScrollView style={styles.mb5}>
+                    <View style={[styles.ph5, styles.mt3]}>
+                        <Text>
+                            {props.translate('twoFactorAuth.scanCode')}
+                            <TextLink href="https://community.expensify.com/discussion/7736/faq-troubleshooting-two-factor-authentication-issues/p1?new=1">
+                                {' '}
+                                {props.translate('twoFactorAuth.authenticatorApp')}
+                            </TextLink>
+                            .
+                        </Text>
+                        <View style={[styles.alignItemsCenter, styles.mt5]}>
+                            <QRCode
+                                url={buildAuthenticatorUrl()}
+                                logo={expensifyLogo}
+                            />
+                        </View>
+                        <Text style={styles.mt5}>{props.translate('twoFactorAuth.addKey')}</Text>
+                        <View style={[styles.mt11, styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween]}>
+                            {Boolean(props.account.twoFactorAuthSecretKey) && <Text>{splitSecretInChunks(props.account.twoFactorAuthSecretKey)}</Text>}
+                            <Button
+                                text={props.translate('twoFactorAuth.copy')}
+                                onPress={() => Clipboard.setString(props.account.twoFactorAuthSecretKey)}
+                                medium
+                            />
+                        </View>
+                        <Text style={styles.mt11}>{props.translate('twoFactorAuth.enterCode')}</Text>
+                    </View>
+                    <View style={[styles.mt3, styles.mh5]}>
+                        <TwoFactorAuthForm />
+                    </View>
+                </ScrollView>
+                <FixedFooter style={[styles.twoFactorAuthFooter]}>
+                    <Button
+                        success
+                        text={props.translate('common.next')}
+                        isDisabled
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/20848

We want the button to be pressed like what we have for the sign-in page and maintain consistency between a similar flow.